### DISCUSSION TITLE
uc mode and remote debugging

### DISCUSSION THREAD

**Discussion by guocity** [seleniumbase/SeleniumBase#3069] [General]
can uc mode work with remote debugging now?

<img width="1139" alt="image" src="https://github.com/user-attachments/assets/52a5e964-a320-4eb9-89a8-80fb404a239d">

**Comment by mdmintz** [comment]
UC Mode already sets the Remote-Debugging port as part of UC Mode internals, so you shouldn't be setting that arg separately while using UC Mode.

**Comment by guocity** [comment]
can you give an example? i can't pass this page with remote-debugging on

**Reply by mdmintz** [reply]
Remote-Debugging is already on for all UC Mode scripts. https://github.com/seleniumbase/SeleniumBase/blob/119ec4bcf38b45d78e77680816aed6d1c24b9b52/seleniumbase/undetected/__init__.py#L141
